src/Augurk/Controllers/V2/ProductsController.cs

```diff
@@ -18,6 +18,7 @@
 using System.Collections.Generic;
```

Comment:
You should probably update the Copyright by adding ,2019 or -2019  to al files you change here.